docs/src/conf.py

```diff
@@ -105,6 +105,14 @@ def setup(app):
 autodoc_member_order = "bysource"
 autodoc_typehints_format = "short"
 
+sphinx_gallery_conf = {
+    "filename_pattern": "/*",
+    "examples_dirs": ["get-started/tutorials/"],
+    "gallery_dirs": ["get-started/tutorials_rendered"],
+    "min_reported_time": 60,
+    "reference_url": {"rascaline": None},
```

Comment:
So, it is possible to have automatic linking of the rascaline reference as well?

Reply:
It should but currently does not work. I will take a another look today.

Reply:
I looked in other libraries and our config looks fine. Maybe we take a look together @Luthaf.